manager/integration/tests/test_node.py

```diff
@@ -2927,8 +2927,7 @@ def test_drain_with_block_for_eviction_if_contains_last_replica_success(client,
     disk_volume_name = 'vol-disk'
     disk_volume = client.create_volume(name=disk_volume_name,
```

Comment:
It seems the `disk_volume` is used to create an extra disk on the host node and is not involved in the subsequent node eviction process. So is the modification necessary?

Reply:
It's from https://github.com/longhorn/longhorn/issues/8753#issuecomment-2172325125, I am fine with either ways.